exercises/luhn/example.el

```diff
@@ -0,0 +1,19 @@
+(require 'seq)
```

Comment:
Would you mind adding the default commentary block back to this file? I dont know that theres a great reason that we have it everywhere other than to avoid agitating the elisp linter, but I'd like to keep consistent in our usage of it.